config/environments/production.rb

```diff
@@ -174,12 +174,6 @@ def call(env)
                                              [
                                                ActionDispatch::RemoteIp::TRUSTED_PROXIES,
                                              ]
-                                           else
```

Comment:
We'll check the assumption that we don't need this with Infra before committing to this